### PR TITLE
refactor a bit to match go style and clean up some datastructures

### PR DESCRIPTION
**Pull request by mhines01**

### CODE REVIEW

**Comment by marcushines** [comment]
ping?

File: daemon/meshnet/handler.go [hunk 1]
```diff
@@ -293,15 +292,14 @@ func (m *Meshnet) AddGRPCWireLocal(ctx context.Context, wireDef *mpb.WireDef) (*
 		PeerIntfID: wireDef.PeerIntfId,
 		PeerPodIP:  wireDef.PeerIp,
 
-		IsReady:       true,
-		HowCreated:    grpcwire.HOST_CREATED_WIRE,
-		CreaterHostIP: "unknown", /*+++todo retrieve host ip and set it here. Needed only for debugging */
+		Originator:   grpcwire.HOST_CREATED_WIRE,
+		OriginatorIP: "unknown", /*+++todo retrieve host ip and set it here. Needed only for debugging */
 
 		StopC:     make(chan bool),
```
Comment (kingshukdev) [inline daemon/meshnet/handler.go:298]: I think it needs to be StopC: make(chan struct{}). 
Will do the merge and will take care of it.

**Review by kingshukdev — COMMENTED** [review]
Changes looks good. I think it needs one more additional change related to stopC. Will take care of it and merge.

File: daemon/meshnet/handler.go (hunk 1 above)
Comment (mhines01) [reply daemon/meshnet/handler.go]: fixed

**Review by apratimmukherjee — APPROVED** [review]

**Comment by kingshukdev** [comment]
Merge done.

**Pull request merged**